lib/generate.js

```diff
@@ -34,3 +36,12 @@ fs.writeFileSync(
     noClaims,
   })
 )
+
+mkdirp(`${__dirname}/../out/blog`)
```

Comment:
Again, this is async and the writeFile below might run first before this is finished.
There is a sync version of this.

Reply:
But more importantly, why not `fs.mkdirSync` ?